website/docs/r/private_network.html.markdown

```diff
@@ -6,7 +6,7 @@ description: |-
   Provides a Vultr private network resource. This can be used to create, read, and delete private networks on your Vultr account.
 ---
 
-# vultr_network
+# vultr_private_network
```

Comment:
other references to vultr_network are still lingering, new pr for this? or just update the other references

Reply:
I'll clean these up while I am here